modules/nf-core/magus/guidetree/main.nf

```diff
@@ -0,0 +1,48 @@
+process MAGUS_GUIDETREE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/magus-msa:0.1.3--pyhdfd78af_0':
+        'biocontainers/magus-msa:0.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.tree"), emit: tree
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    magus \\
+        -np $task.cpus \\
+        -i $fasta \\
+        -o ${prefix}.tree \\
+        --onlyguidetree TRUE \\
```

Comment:
is `--onlyguidetree TRUE` a required option? Is there no use for the module without that flag?

Reply:
The guidetree module is made specifically to compute a guidetree with the align module handling the alignment, so I think there would not be many uses.
In theory the user could pass `--onlyguidetree FALSE` in ext.args, overwriting this I guess, but it would make more sense to just call the right module in the first place.

Reply:
I'm not sure splitting this is the best option since they are the same tool (not even a different subtool)

Reply:
Ah sorry, missed this earlier.
Maybe having the same functionality using a single module and multiple optional outputs would be cleaner.
This was done to maintain consistency with how we are handling it for other MSA tools that can do both, e.g. [ClustalO](https://github.com/nf-core/modules/tree/master/modules/nf-core/clustalo) or [FAMSA](https://github.com/nf-core/modules/tree/master/modules/nf-core/famsa).

Reply:
That is a fair point, yet the guidetree has different input channels compared to the align module, that is why in the other modules we kept it separate (famsa and clustalo)

Reply:
This seems reasonable to keep separate in this case.